src/coreclr/inc/clrconfigvalues.h

```diff
@@ -670,6 +666,9 @@ RETAIL_CONFIG_DWORD_INFO_EX(EXTERNAL_PreferredVectorBitWidth,      W("PreferredV
 #if defined(TARGET_LOONGARCH64)
 //TODO: should implement LoongArch64's features.
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableHWIntrinsic,            W("EnableHWIntrinsic"),         0, "Allows Base+ hardware intrinsics to be disabled")
+#elif defined(TARGET_IOS) || defined(TARGET_TVOS) || defined(TARGET_MACCATALYST)
+// Always disabled on Apple mobile
```

Comment:
Should this be a TODO with a link to an issue?

We do want to have min baseline of HW intrinsics enabled on iOS for good perf. It will require changing crossgen2 policies: disable optimistic intrinsic sets, and make sure that all HW intrinsics leaf methods in CoreLib have R2R code that the interpreter can call.

Reply:
Hm, right - this is not what I want. This was a poor attempt at reconciling what was set in the test that is currently interpreter only. I'll switch back to having that test explicitly configure. And make sure we do -optimistic for crossgen2 in a follow-up after.